README.md

```diff
@@ -28,13 +28,13 @@ First you'll need to make sure you have a full development environment set up:
 #### (macOS)
 
 ```
-brew install cmake openssl tor
+brew install cmake openssl tor ncurses
```

Comment:
Is this needed?

Reply:
I thought to mention it as a pre-requisite as sometimes there is contention about which version is correct, as per https://stackoverflow.com/questions/56622042/clang-on-macos-fails-linking-lmenu-from-ncurses where an explicit `brew install ncurses` was required. Maybe @sdbondi can comment on this as well?

Reply:
This is a dev dependency - so it shouldn't be needed for release builds.

I'm actually happy to remove the pingpong example that depends on this, it has become unimportant for testing the network since testnet

Reply:
I could not do a release build on a fresh Ubuntu installation without these dependencies installed, hence this PR.
See registered issue #2274 to remove the pingpong example as suggested.

Reply:
Thats not to say that this must be done in this PR